.github/workflows/integration-tests-vm-type.yml

```diff
@@ -113,24 +118,39 @@ jobs:
           if [[ '${{ contains(github.event.pull_request.labels.*.name, 'integration-tests-trace-logging') }}' == 'true' ]]; then
             echo "COLLECTOR_LOG_LEVEL=trace" | tee -a "$GITHUB_ENV"
           fi
-
+          
           if [[ '${{ inputs.vm_type }}' =~ s390x ]]; then
             {
               echo "IC_API_KEY=${{ secrets.IBM_CLOUD_S390X_API_KEY }}"
               echo "IC_REGION=ca-tor"
             } >> "${GITHUB_ENV}"
           fi
 
+          if [[ '${{ inputs.vm_type }}' =~ ppc64le ]]; then
+            {
+              echo "IC_API_KEY=${{ secrets.IBM_CLOUD_POWER_API_KEY }}"
+              echo "IC_REGION=osa"
+              echo "IC_ZONE=osa21"
+            } >> "${GITHUB_ENV}"
+          fi
+
           if [[ '${{ inputs.vm_type }}' =~ rhel ]]; then
             {
               echo "REDHAT_USERNAME=${{ secrets.REDHAT_USERNAME }}"
               echo "REDHAT_PASSWORD=${{ secrets.REDHAT_PASSWORD }}"
             } >> "${GITHUB_ENV}"
           fi
+#added workaround for ssh connection issue with power vm from ubuntu machine , changing mtu works successfully ssh to power vms      
+      - name: Set MTU for Power VMs
+        if: ${{ contains(inputs.vm_type, 'ppc64le') }}
+        run: sudo ifconfig eth0 mtu 1000 up  
+  
       - name: Create VMs
         if: ${{ ! inputs.run-benchmarks }}
         run: |
           make -C "${{ github.workspace }}/ansible" create-ci-vms
+        env:
+          IBM_CLOUD_POWER_SSH_PUBLIC_KEY: ${{ secrets.IBM_CLOUD_POWER_SSH_PUBLIC_KEY }}
```

Comment:
Can we move this to the script on line 130 instead of added to the create vm step?